druid/src/widget/textbox.rs

```diff
@@ -104,7 +104,7 @@ impl TextBox {
             .unwrap_or(FontFamily::SYSTEM_UI);
 
         piet_text
-            .new_text_layout(&text.to_string())
+            .new_text_layout(text.to_string())
```

Comment:
```suggestion
            .new_text_layout(text)
```

Reply:
going to ignore my own suggestion so we don't need to run CI again, this will get changed in #1182 anyway.